.github/workflows/manual-deploy.yml

```diff
@@ -0,0 +1,39 @@
+name: Deploy to AKS (Manual)
+concurrency: manul_deploy_${{ github.event.inputs.environment }}
+
+on:
+  workflow_dispatch:
+    inputs:
+      docker-image-tag:
+        description: 'Docker image tag'
+        required: true
+        default: 'latest'
+      environment:
+        description: 'Target environment'
+        required: true
+        default: 'development'
+
+jobs:
+  manual-deploy:
```

Comment:
The job environment must be specified so it can find the right AZURE_CREDENTIALS